config/maintainers.json

```diff
@@ -80,6 +80,16 @@
       "link_url": "null",
       "avatar_url": "https://s.gravatar.com/avatar/e5021d7393c10095cc2baf9a63ba40b4",
       "bio": "Aspiring polyglot who spent far too many years confined to C# to let it go to waste."
+    },
+    {
+      "github_username": "shaynuhcon",
+      "alumnus": false,
+      "show_on_website": true,
```

Comment:
Welcome to the party @shaynuhcon! I look forward to working with you in the future :)

Since you're a new maintainer can you please set this value to false for now. Once you're up-to-speed for contributing then we'll add you to the website.

Reply:
For sure. Wasn't really sure what to put there but it has been updated now!